### PR TITLE
Fix issue #3 and also remove redundant spaces in connect/disconnect logs that fucking irritated me

### PR DESCRIPTION
**Pull request by Butterroach**
\+ add a single new line at the end of main.py and add extra dependency (psutil)

:3

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -11,6 +12,9 @@
 
 @sio.event
 def imgcaption(sid, data):
+    if psutil.virtual_memory().available / 1048576 < 2000:  # check if available ram is less than 2gb (2GB != 2048MB btw 2GiB == 2048MiB GB != GiB MB != MiB you're just an idiot)
```
Comment (artifishvr) [inline main.py:15]: why did you add that comment

Comment (Butterroach) [reply main.py:15]: 3:03 AM

**Comment by artifishvr** [comment]
looks great sure hope you tested it

**Comment by Butterroach** [comment]
i didnt :trollface: (i only tested the check)
